api/campaigns/class-maybe-show-campaign.php

```diff
@@ -325,10 +325,10 @@ public function should_popup_be_shown( $client_id, $popup, $settings, $referer_u
 			$frequency_reset   = 'month';
 		}
 		if ( 'preset_1' === $frequency ) {
-			$frequency_max     = 5;
-			$frequency_start   = 3;
+			$frequency_max     = 0;
+			$frequency_start   = 0;
 			$frequency_between = 3;
-			$frequency_reset   = 'month';
+			$frequency_reset   = 'day';
```

Comment:
Just noting that this will affect any prompts already set to this preset-frequency. But since the setting was experimental up till now, this is acceptable.

Reply:
This is true, but I don't believe any site was actually using the preset as it was more or less for our own convenience during beta testing.